lib/pages/send_view/send_view.dart

```diff
@@ -881,6 +881,48 @@ class _SendViewState extends ConsumerState<SendView> {
     }
   }
 
+  String _getSendAllTitle(bool showCoinControl, Set<UTXO> selectedUTXOs) {
+    if (showCoinControl && selectedUTXOs.isNotEmpty) {
+      return "Send all selected";
+    }
+
+    return "Send all ${coin.ticker}";
+  }
+
+  Amount _selectedUtxosAmount(Set<UTXO> utxos) => Amount(
+        rawValue:
+            utxos.map((e) => BigInt.from(e.value)).reduce((v, e) => v += e),
+        fractionDigits: ref.read(pWalletCoin(walletId)).fractionDigits,
+      );
+
+  Future<void> _sendAllTapped(bool showCoinControl) async {
+    final Amount amount;
+
+    if (showCoinControl && selectedUTXOs.isNotEmpty) {
+      amount = _selectedUtxosAmount(selectedUTXOs);
+    } else if (isFiro) {
```

Comment:
My only qualm so far is this else if, I just need to read over it a few more times to grok if this is going to be an issue or not (probably not)